tools/mason/MasonExternal.chpl

```diff
@@ -18,7 +18,12 @@
  * limitations under the License.
  */
 
-config const spackVersion = "releases/v0.11.2";
+const spackVersion = '0.14.2';
```

Comment:
Let's put the Chapel version number in here to remind us to update this next release:

```suggestion
/* Version as of Chapel 1.22 - to be updated each release */
const spackVersion = '0.14.2';
```